src/Tizen.NUI.Components/Theme/DefaultThemeCommon.cs

```diff
@@ -284,7 +285,9 @@ public Theme Create()
                 },
                 ValueIndicatorImage = new ImageViewStyle()
                 {
-                    Size = new Size(43, 40),
+                    SizeHeight = 40,
+                    WidthResizePolicy = ResizePolicyType.FitToChildren,
```

Comment:
When an indicator image has a fixed height but variable width, can it be ellipse in some cases?

Reply:
Yes, the default value is just drawing the border line and `ValueIndicatorImage` property has `ValueIndicatorText` as a child.
So, the width could be resized depending on the text length. :)
Thank you for your review!